src/app/components/new-goal/new-goal.component.ts

```diff
@@ -8,17 +10,19 @@ import {MatDatepickerInputEvent} from '@angular/material';
 })
 export class NewGoalComponent implements OnInit {
   goalName: string;
+  goalWorkload: number;
```

Comment:
Add documentation to this since it's not very clear.

Reply:
ok